active/operators/bort/generate_job_id.py

```diff
@@ -24,8 +24,16 @@ def generate_job_id(**kwargs):
     # get last update date from source since last successful run
     # then pull record id with new update date from source
     job_start_date = datetime.now().strftime(common_variables.DT_FORMAT)[:-3]
+    bortdates = []
 
+    blob_job_queue = _get_blobs_since_date(date=last, job_state=common_variables.JOB_COMPLETE)
 
+    #check the queue for any new jobs that need to be scheduled
+    if blob_job_queue is None:
+      exit()
```

Comment:
I think it'd be good to add some kind of logging or print statement here so that it's clear that the exit is expected and there are no available notes to run

Reply:
ditto as above - just saw the update in the other commit